### PR TITLE
Upgrade dependencies and build set up for CWP 1

### PR DESCRIPTION
**Pull request by maxime-rainville**
# Parent issue
* https://github.com/silverstripeltd/product-issues/issues/48

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -29,7 +26,12 @@
     "eslint-plugin-import": "^2.2.0",
     "eslint-plugin-jsx-a11y": "^3.0.2",
     "eslint-plugin-react": "^6.9.0",
+    "laravel-mix": "^4",
+    "path": "^0.12.7",
+    "sass": "^1.23.3",
     "sass-lint": "^1.10.2",
-    "webpack": "^2"
+    "sass-loader": "^7.3.1",
+    "vue-template-compiler": "^2.6.10",
```
Comment (Cheddam) [inline package.json:34]: Did Laravel Mix install this? I don't think it's necessary for this project (Vue itself isn't being pulled in).

File: .travis [hunk 2]
```diff
@@ -0,0 +1,16 @@
+language: node_js
```
Comment (Cheddam) [inline .travis]: Does this need to be named `.travis.yml`?

File: package.json (hunk 1 above)
Comment (maxime-rainville) [reply package.json:34]: Yes

File: .travis (hunk 2 above)
Comment (maxime-rainville) [reply .travis]: Fixed.

**Pull request merged**